pootle/apps/pootle_app/management/commands/contributors.py

```diff
@@ -87,3 +139,13 @@ def list_contributions(self, contribs, sort_by):
 
         for line in out:
             self.stdout.write(line)
+
+    def list_only_emails(self, units):
+        nobody_user = User.objects.get_system_user()
```

Comment:
I presume this should be `nobody_user = User.objects.get_nobody_user()` instead.